src/Integration.UnitTests/Connection/ConnectControllerTests.cs

```diff
@@ -115,19 +120,57 @@ public void ConnectionController_ConnectCommand_Status()
             testSubject.ConnectCommand.CanExecute().Should().BeFalse("Busy");
 
             // Case 4: no connection, not busy
+            this.host.TestStateManager.IsConnected = false;
             this.host.VisualStateManager.IsBusy = false;
 
             // Act + Assert
             testSubject.ConnectCommand.CanExecute().Should().BeTrue("No connection and not busy");
         }
 
         [TestMethod]
+        public void ConnectionController_ConnectCommand_SolutionNotFully_Open_Status_AlwaysFalse()
```

Comment:
New test covering the permutations of the command status check when the solution is not fully loaded.